src/EditorFeatures/DiagnosticsTestUtilities/CodeActions/AbstractCodeActionTest.cs

```diff
@@ -85,8 +85,8 @@ public abstract partial class AbstractCodeActionTest : AbstractCodeActionOrUserD
                 return null;
 
             var fixAllState = new FixAllState(fixAllProvider, document, selectionSpan, provider, optionsProvider, scope, originalCodeAction);
-            var fixAllContext = new FixAllContext(fixAllState, CodeAnalysisProgress.None, CancellationToken.None);
-            return await fixAllProvider.GetFixAsync(fixAllContext).ConfigureAwait(false);
+            var fixAllContext = new FixAllContext(fixAllState, CancellationToken.None);
```

Comment:
importantly, 'progress' is not part of hte 'context'.  The context represents immutable state about the world explaining what the fix all needs to do.  Ambient state about that particular part of the operation needs to be passed in as arguments to that actual operation.

Note: we are unfortunately inconsistent here (due to back compat) on this aspect.  Specifically, the FixAllContext unfortunately passed along a CancellationToken.  This has actually been a problem in the past with how that is captured.  So keeping these explicitly separate is something we have a strong desire to do.

That said, there's an open question if we should then just do the same for the cancellation token as well.  in other words, the apis updated here would be updated to go from `XXX(FixAllContext context)` to `XXX(FixAllContext context, IProgress<CodeAnalysisProgress> progress, CancellationToken cancellationToken)`.

@mavasani Do you have any thoughts on this?